packages/cursorless-vscode-e2e/src/suite/fixtures/recorded/languages/markdown/chuckItem2.yml

```diff
@@ -0,0 +1,28 @@
+languageId: markdown
+command:
+  version: 5
+  spokenForm: chuck item
+  action: {name: remove}
+  targets:
+    - type: primitive
+      modifiers:
+        - type: containingScope
+          scopeType: {type: collectionItem}
+  usePrePhraseSnapshot: true
+initialState:
+  documentContents: |-
+    - aaa
+      - bbb
+      - ddd
+  selections:
+    - anchor: {line: 1, character: 5}
+      active: {line: 1, character: 5}
+  marks: {}
+finalState:
+  documentContents: |-
+    - aaa
+
```

Comment:
so is the idea to handle this one as part of [#855](https://github.com/cursorless-dev/cursorless/issues/855)?

Reply:
Maybe? For now this is just an a improvement of what we had, but not perfect.

Reply:
Seems fine